pymola/Modelica.g4

```diff
@@ -535,8 +545,8 @@ WS  :   [ \r\n\t]+ -> skip ; // toss out whitespace
 //=========================================================
 fragment Q_IDENT : '\'' ( Q_CHAR | S_ESCAPE)+;
 fragment NONDIGIT : [_a-zA-Z];
-fragment S_CHAR : [A-Za-z\u0000-\u00FF];
-fragment Q_CHAR : NONDIGIT | DIGIT | [!#$%&()*+,-./:;<>=?@[\]^{}|! ];
+fragment S_CHAR : [\u0000-\u00FF];
+fragment Q_CHAR : NONDIGIT | DIGIT | [!#$%&()*+,-./:;<>=?@[\]^{}| ];
```

Comment:
This is a fix for a warning I was getting from antlr about redundant characters.